File: tests/Fortran/binaire.f90
program binaire
  integer :: a = 16330

  a = affiche_binaire(a)

contains

  function affiche_binaire (n) result(res)
    integer :: n
    integer :: res
    integer :: dividende, mod_2
    integer :: i

    do i = 0, 15
      dividende = n/2
      mod_2 = n - 2*dividende
      print *, mod_2
      n = dividende
    end do

    res = 0

  end function affiche_binaire

end program binaire

File: tests/C/binaire.c
#include <stdio.h>
int affiche_binaire(int n){


    int res;
    int dividende, mod_2;
    int i;

    for (i = 0; i < 15; i=i + 1) {
        dividende = n / 2;
        mod_2 = n - 2 * dividende;
        printf("%d", mod_2);
        n = dividende;
    }    res = 0;



    return res;
}
// binaire
void main(void){
    int a = 16330;

    a = affiche_binaire(a);




}